lib/collection/src/collection/mod.rs

```diff
@@ -141,6 +143,7 @@ impl Collection {
             this_peer_id,
             path: path.to_owned(),
             snapshots_path: snapshots_path.to_owned(),
+            snapshot_storage_manager: Self::create_snapshot_storage_manager(snapshots_path),
             channel_service,
```

Comment:
I think that in the future snapshots_path could be remove so we will avoid info duplication, the path can be retrieved from the snapshot_storage_manager.